coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -7927,15 +7927,18 @@ def test_constant_pad_3d(self, compute_unit, backend):
 
 class TestMaskedFill(TorchBaseTest):
     @pytest.mark.parametrize(
-        "compute_unit, backend",
-        itertools.product(compute_units, backends),
+        "compute_unit, backend, value",
+        itertools.product(
+            compute_units,
+            backends,
+            [10.0, 0],
+        ),
     )
-    def test_masked_fill(self, compute_unit, backend):
+    def test_masked_fill(self, compute_unit, backend, value):
```

Comment:
By default, the testing infra is producing the fp32 input only.
However, in this change, we are missing the case that input is `int32`, in which the `value` might be downcast.
Please refer to the `converter_input_type` argument,
to parametrize the input dtype as well.
Also, it would be better to test the value of `[10.3, 7]` instead of `[10.0, 0]`.
With `10.3`, we can test the case that `10.3` be downcasted to `int`,
and `7` is in generally a better choice than `0` for the testing purpose,
given that `0` could potentially be in some default value haha 😄

Reply:
https://github.com/apple/coremltools/blob/main/coremltools/converters/mil/frontend/torch/test/test_torch_ops.py#L7030
An example of using `converter_input_type`